viewing_party/main.py

```diff
@@ -0,0 +1,221 @@
+
+def create_movie(title, genre, rating): 
+    movie = dict();
+    if title:
+        movie['title'] = title;
+    else:
+        return None
+    if genre:
+        movie['genre'] = genre;
+    else:
+        return None
+    if rating:
+        movie['rating'] = rating;
+    else:
+        return None
+    return movie
```

Comment:
You can clean this up using a compound conditional and dictionary literal:

```suggestion
    if title and genre and rating:
        return {
            "title": title,
            "genre": genre,
            "rating": rating
        }
    else:
        return None
```

This way you don't need to repeat the `return None` lines.  😄